packages/next/server/dev/static-paths-worker.ts

```diff
@@ -49,7 +49,7 @@ export async function loadStaticPaths({
 
   // update work memory runtime-config
   require('../../shared/lib/runtime-config').setConfig(config)
-  setHttpAgentOptions(httpAgentOptions)
+  setHttpClientAndAgentOptions({ httpAgentOptions })
```

Comment:
Looks like this is missing the `enableUndici` config so would cause it to be used in the parent but not the worker.